src/app/donation-start/donation-start-form/donation-start-form.component.html

```diff
@@ -493,6 +493,10 @@ <h3 class="b-rt-0 b-m-0 b-bold"><span class="span-hr"></span>How does Big Give u
                     Please note that you might continue to receive communications from the charity if you have already shared your details with them via other methods.
                   </p>
                 }
+                @if (marketingGroup.value.optInCharityEmail === true) {
+                  <p aria-live="polite" class="no-margin-top b-rt-sm b-grey">
+                    As you are providing personal information directly to {{ campaign.charity.name }}, please make sure you read and agree to their privacy statement.</p>
```

Comment:
It's not really direct, is it? We're passing the info on.
```suggestion
                    As you are providing personal information to {{ campaign.charity.name }}, please make sure you read and agree to their privacy statement.</p>
```